src/tox_travis/envlist.py

```diff
@@ -43,12 +45,15 @@ def autogen_envconfigs(config, envs):
     config.distshare = reader.getpath("distshare", distshare_default)
     reader.addsubstitutions(distshare=config.distshare)
 
-    # Create the undeclared envs
-    for env in envs:
+    try:
+        make_envconfig = tox.config.ParseIni.make_envconfig  # tox 3.4.0+
+    except AttributeError:
```

Comment:
Since the try/except is what's making codecov complain, what do you think about just doing things the way of the latest version of tox, since that's what's going to be installed? Or does that not work with Python 2.6? If the latest version of tox doesn't work with Python 2.6, then I'm actually inclined to drop support for Python 2.6 in this library, rather than keep it around and worry about backward compatibility with an old version of tox. I don't think it buys us anything to keep backward compatible with old versions of tox.

Reply:
> Or does that not work with Python 2.6?

Yes.

I agree with dropping it.

But I also think that we can assume that the code is hit via py26 currently, although coverage reporting does not work (appears to be a problem with coverage/coverage_pth, not codecov), but not worth investigating.

As for a new release I think it's better to not necessarily require an updated tox also etc - does not really seem necessary, but it is your call.